index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Microverse Portfolio Page</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <header>
+      <nav id="toolbar">
+        <a href="#" id="logo">My Logo</a>
+        <div id="menu-bars-container">
+          <div id="menu-bars">
+            <div class="menu-bar"></div>
+            <div class="menu-bar"></div>
+            <div class="menu-bar"></div>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <section id="headline-container">
+      <h1 class="headline-header">
+        <span> I’m Hafed <br />Glad to see you! </span>
+      </h1>
+      <p class="headline-text">
+        I’m a software developer! I can help you build a product, feature or
+        website. Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <span id="connect">Let's Connect</span>
```

Comment:
- Kindly consider adding an anchor tag `<a>` tag to your "Let's Connect" link. The "a" tag will let you connect to other sections of the website, using href. 🤗